external/libcxx/iostream.cxx

```diff
@@ -58,7 +58,11 @@ static mbstate_t mb_wcerr;
 _ALIGNAS_TYPE (ostream)  _LIBCPP_FUNC_VIS char clog[sizeof(ostream)];
 _ALIGNAS_TYPE (wostream) _LIBCPP_FUNC_VIS char wclog[sizeof(wostream)];
 
+#ifdef CONFIG_BINFMT_CONSTRUCTORS
```

Comment:
I think we need to add contidions about loadable build because both are declared as static variable for constructor.